api/odigos/v1alpha1/instrumentation_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/odigos-io/odigos/k8sutils/pkg/workload"
+)
+
+// Instrumentation configures an application for auto-instrumentation.
+// +genclient
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:metadata:labels=metadata.labels.odigos.io/config=1
+// +kubebuilder:metadata:labels=metadata.labels.odigos.io/system-object=true
```

Comment:
Thanks for clarifying, I wasn't sure what these were for. @RonFed said that they were used during uninstall, if that's the case then do you think we would want to remove these objects if someone is uninstalling Odigos?

Reply:
Testing this out, I do think this CRD should be removed when you run `odigos uninstall`, so imo we should keep these labels but maybe I'm misunderstanding?

Reply:
They are set by the CLI [here](https://github.com/odigos-io/odigos/blob/5f3b442f06c227a89dde596d72b1e57cf651dc10/cli/pkg/kube/client.go#L186).

The CRDs will be removed when running `odigos uninstall`.

If someone installs odigos with helm and them uninstall with cli - it will fail, but the fix is more complicated than adding these labels

Reply:
Talked offline and removed these labels from the CRD, thanks for clearing it up @blumamir !